diff --git a/dist/index.js b/dist/index.js
index 93cf348bd441f278b8005dbdead14be7ac7ba326..9b86ee452365e8c07da77b60ef1bec9b7989dc8b 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -762,11 +762,13 @@ const ae = "embeddedJson", _s = ["json", "jsonl"], vs = ["json", "json5", "jsonc
   ).join(""), c = d.match(/^\s+/)?.[0] ?? "", p = d.match(/\s+$/)?.[0] ?? "", f = d.slice(
     c.length,
     -p.length || void 0
-  ), b = S(n, t), _ = await e(f, {
+  );
+  const b = S(n, t), _ = await e(qr(f), {
     ...s,
     parser: s.embeddedMarkdownParser ?? "markdown",
     __inJsTemplate: !0
-  }), o = A(_, E, b);
+  }), o = A(_, E, b, !0);
+
   if (s.preserveEmbeddedExteriorWhitespaces?.includes(i))
     return F([
       "`",
